src/views/Dashboard.vue

```diff
@@ -7,31 +7,87 @@
     <div class="buton-wrap">
       <button @click="logout">ログアウトする</button>
     </div>
+
+    <div class="users-wrap">
+      <table>
+        <thead>
+          <th colspan="3">ユーザー名</th>
+        </thead>
+        <tbody>
+          <tr v-for="(userData, index) in users" :key="index">
+            <td>
+              {{ userData.name }}
+            </td>
+            <td>
+              <button @click="walletToggle(index)">wallteを見る</button>
+            </td>
+            <td>
+              <button @click="sendToggle(index)">送る</button>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+    <div v-if="walletShow">
+      <walletModal @walletClose="walletToggle()" :users="users" :clickIndex="clickIndex" />
+    </div>
+    <div v-if="sendShow">
+      <sendModal @sendClose="sendToggle()" :user="user" />
+    </div>
   </div>
 </template>
 
 <script>
 import store from '../store';
+import walletModal from '@/components/walletModal.vue';
+import sendModal from '@/components/sendModal.vue';
 
 export default {
   name: 'Login',
   data() {
-    return {};
+    return {
+      walletShow: false,
+      sendShow: false,
+      clickIndex: '',
+    };
   },
   created() {
     store.dispatch('checkLogin');
+    store.dispatch('usersData');
   },
   computed: {
     user: {
       get() {
         return store.getters.user;
       },
     },
+    users: {
+      get() {
+        return store.getters.users;
+      },
+    },
   },
   methods: {
     logout() {
       store.dispatch('logout');
     },
+    walletToggle(index) {
+      this.walletShow = !this.walletShow;
+      if (index !== undefined) {
```

Comment:
indexが引数に渡ってこない時、という意味でこの条件を記載していますか？
もしそうならどのような時に発生しますか？
indexがundefinedの時があるのかが気になりました。

ちなみに同じ条件なら以下のように記載できます。
```suggestion
      if (index) {
```

Reply:
modalを閉じる際の処理で必要なかったのでindexを渡さないようにしていたのですが。
確かに、コードを見た人へ不要な場合があるかどうかを考えさせてしまう可能性があると思ったので今回削除いたしました。